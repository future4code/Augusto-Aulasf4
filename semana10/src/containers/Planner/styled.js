import styled from 'styled-components'

export const PlannerContainer = styled.div`
  display:grid;
  grid-template-columns: repeat(7,1fr);
  min-height:100vh;
`

export const IncludeButton = styled.button`
background: lightgreen;
`

export const InputsContainer = styled.div`
  grid-column:1/-1;
  height:5vh;
  display:flex;
  width:100%;
  background: lightpink;
  justify-content:space-evenly;
`

export const Title = styled.h3`
  background: lightsteelblue;
  color:white;
  text-align:center;
  margin:0;
  border: 2px black solid; 
  height:5vh; 
`

export const Day = styled.h3`
  background: lightcyan;
  color:gray;
  text-align:center;
  margin:0;
  padding:20px;
  border: 1px black solid;
  min-height:90vh;
`

export const Activity = styled.span`
  border: 1px black solid;
  display:flex;
  flex-direction:column;
  margin:10px;
`