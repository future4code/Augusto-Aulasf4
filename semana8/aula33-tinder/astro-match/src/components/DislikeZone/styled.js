import styled, { css, keyframes } from 'styled-components'

export const Disliked = styled.div`
width: 20%;
height:100%;
background: red;
position:absolute;
left:0;
`