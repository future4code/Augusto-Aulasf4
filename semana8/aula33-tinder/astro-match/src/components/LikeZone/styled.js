import styled, { css, keyframes } from 'styled-components'

export const Liked = styled.div`
width: 2%;
height:100%;
background: green;
position:absolute;
right:0;

`