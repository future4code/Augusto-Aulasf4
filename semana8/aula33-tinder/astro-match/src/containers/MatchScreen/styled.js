import styled, { keyframes } from 'styled-components'
import Icon from '@mdi/react'

export const MatchIcon = styled(Icon)`
	display: block;
	fill: #4AA397;
`

export const swipeRightToInit = keyframes`
  from {
    opacity: 0;
	  transform:scale(0) translate(0) rotate(0);
  }

  to {
    opacity: 1;
	  transform: scale(1) translate(0) rotate(360deg);
  }
`;
export const swipeLeftToClose = keyframes`
  from {
    opacity: 1;
	  transform:scale(1) translate(0) rotate(0);
  }

  to {
    opacity: 0;
	  transform: scale(0) translate(0) rotate(360deg);
  }
`;

export const MatchContainer = styled.div`
animation: ${props=>props.animation} 1s cubic-bezier(0.215, 0.61, 0.355, 1);
animation: ${props => props.closeWindowWithAnimation} 0.6s ;
`

export const List = styled.ul`
  padding: 0 10px;
`

export const DivOver = styled.div`
height:520px;
overflow-y:scroll;
`

export const ListItem = styled.li`
  position: relative;
  height: 70px;
  padding: 10px;
  display: flex;
  list-style: none;
  width: 100%;
  transition: 0.2s;
  cursor: pointer;
  
  :hover {
    background: #eee;
  }
  
  :last-child {
    border: unset;
  }
  
  :active {
			:before {
				height: 100%;
				width: 100%;
				top: 0;
    		left: 0;
    		position: absolute;
    		background-color: rgba(0, 0, 0, 0.1);
    		content: "";
			}
   }
`

export const Avatar = styled.div`
  height: 100%;
  width: 50px;
  margin-right: 10px;
  border-radius: 50%;
  background: url(${props => props.src});
  background-position: center center;
  background-size: 100%;
  background-repeat: no-repeat;
`

export const ListText = styled.p`
  user-select: none;
`
