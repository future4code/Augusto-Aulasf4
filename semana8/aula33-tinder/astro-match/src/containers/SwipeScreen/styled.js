import styled, {keyframes} from 'styled-components'
import Icon from '@mdi/react'

export const OptionButton = styled.button`
   border-radius: 50%;
   width: 80px;
   height: 80px;
   border: 1px solid ${(props) => props.option === 'like' ? 'green' : 'red'};
   color: ${(props) => props.option === 'like' ? 'green' : 'red'};
   font-size: 50px;
   transform: scale(0.7);
   transition: 0.2s;
   position: relative;
   box-shadow: 0 0 15px 0 rgba(205, 205, 205, 0.73);
   overflow: hidden;
   
   :focus {
   		outline: none;
   }
   
   :hover {
    background-color: ${props => props.option === 'like' ? 'green' : 'red'};
    color: white;
    transform: scale(0.8)
   }
   
   :active {
			:before {
				height: 100%;
				width: 100%;
				top: 0;
    		left: 0;
    		position: absolute;
    		background-color: rgba(0, 0, 0, 0.5);
    		content: "";
			}
   }
`

export const ContentWrapper = styled.div`
	padding: 20px;
	padding-bottom: 0;
	display: flex;
	flex-direction: column;
	flex: 1;
	justify-content: flex-end;
`

export const swipeRightToInit = keyframes`
  from {
    opacity: 0;
	  transform:scale(0) translate(0) rotate(0);
  }

  to {
    opacity: 1;
	  transform: scale(1) translate(0) rotate(-360deg);
  }
`;
export const swipeLeftToClose = keyframes`
  from {
    opacity: 1;
	  transform:scale(1) translate(0) rotate(0);
  }

  to {
    opacity: 0;
	  transform: scale(0) translate(0) rotate(-360deg);
  }
`;


export const SwipeScreenWrapper = styled.div`
	display: flex;
	flex-direction: column;
	height: 100%;
  animation: ${props => props.closeWindowWithAnimation} 0.6s ;
`


export const MatchIcon = styled(Icon)`
	display: block;
	fill: #753192;
`

export const ButtonsWrapper = styled.div`
  display: flex;
  justify-content: space-evenly;
  align-items: center;
  padding: 10px 0;
`
