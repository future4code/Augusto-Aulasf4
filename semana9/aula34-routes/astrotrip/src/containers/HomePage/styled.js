import styled from 'styled-components'

export const HomeContainer = styled.div`
    font-family:"Comic Sans MS", cursive, sans-serif;
    display:flex;
    flex-direction:column;
    justify-content:center;
    align-items:center;
    width:100%;
    min-height:100vh;
    background:url('https://uploads-ssl.webflow.com/5d640f4558306be99cf47a0e/5d7ebf13cb34e44fada5540c_estrelas_fundo4.png'), linear-gradient(360deg, #3068d0, #692b90 50%, #120f33 80%, #050505);
`

export const ButtonSpace = styled.button`
    color:white;
    font-weight:bold;
    padding:10px;
    font-size:25px;
    background: linear-gradient(#385FC6,#662A8D);
    border-radius:10px;
    border:3px solid black;
    cursor:pointer;

    :focus{
        outline:none;
    }

    :active{
        position:relative;
        top:2px;
    }
`

export const ContentContainer = styled.div`
    display:flex;
    justify-content:space-evenly;
    align-items:center;
    width:60%;
    min-height:70vh;
    background:rgba(255,255,255,0.8);
    border-radius:50px;
    border:5px solid black;
`

export const ImgLogo = styled.img`
    width:30%;
`

export const TextArea = styled.div`
    display:flex;
    flex-direction:column;
    justify-content:space-evenly;
    width:30%;
    min-height:30vh;
`

export const ButtonArea = styled.div`
    display:flex;
    justify-content:space-between;
    width:90%;
`

