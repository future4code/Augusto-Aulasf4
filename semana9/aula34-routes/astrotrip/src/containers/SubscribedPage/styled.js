import styled from 'styled-components'

export const SubscriberWrapper = styled.div`
    display:flex;
    font-family:"Comic Sans MS", cursive, sans-serif;
    flex-direction:column;
    justify-content:space-evenly;
    align-items:center;
    width:100%;
    min-height:100vh;
    background:url('https://uploads-ssl.webflow.com/5d640f4558306be99cf47a0e/5d7ebf13cb34e44fada5540c_estrelas_fundo4.png'), linear-gradient(360deg, #3068d0, #692b90 50%, #120f33 80%, #050505);
`;

export const HeaderContent = styled.div`
    display:flex;
    align-items:center;
    justify-content:space-around;
    width:100%;
    height:80px;
    background: white;
    margin-top: 3%;
    border:2px solid black;
`;

export const BodyContent = styled.div`
    flex:1;
    width:100%;
    display:flex;
`;

export const CandidateContent = styled.div`
    display:flex;
    flex-direction:column;
    align-items:center;
    justify-content:center;
    width:50%;
    height:80vh;
    position: relative;
`

export const ButtonContent = styled.div`
    display:flex;
    justify-content:space-evenly;
    width:100%;
    margin:10%;
`

export const SettingContent = styled.div`
    display:flex;
    flex-direction:column;
    align-items:center;
    justify-content:flex-end;
    width:50%;
    height:80vh;
`;

export const MenuTitle = styled.h2`
    display:flex;
    flex-direction:column;
    background:white;
    color:#7b1fa2;
    width:100%;
    text-align:center;
`;

export const MenuItems = styled.p`
    display:flex;
    flex-direction:column;
    color:white;
    width:100%;
    font-size:130%;
    text-align:center;
    padding:5px;
    margin:0;
    cursor:pointer;
    border-bottom:1px solid white;
`;

export const ButtonMenu = styled.button`
    color:#7b1fa2;
    font-weight:bold;
    padding:10px;
    font-size:25px;
    background: white;
    border-radius:10px;
    border:3px solid black;
    cursor:pointer;
    box-shadow: 0 2px 5px rgba(0,0,0,0.5);
    margin-top:5%;
    :focus{
        outline:none;
    }

    :active{
        position:relative;
        top:2px;
    }
`



export const ContentContainer = styled.div`
    display:flex;
    flex-direction:column;
    align-items:center;
    width:95%;
    min-height:90vh;
    background:rgba(255,255,255,0.8);
    border-radius:30px;
    border:5px solid black;
`