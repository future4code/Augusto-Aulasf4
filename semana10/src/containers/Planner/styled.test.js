import React from "react";
import renderer from "react-test-renderer";
import { PlannerContainer, InputsContainer, Title, Day, Activity, IncludeButton } from './styled'

describe("Styleds", () => {

    it("PlannerContainer", () => {
      const tree = renderer.create(<PlannerContainer />).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it("IncludeButton", () => {
      const tree = renderer.create(<IncludeButton />).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it("InputsContainer", () => {
      const tree = renderer.create(<InputsContainer />).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it("Title", () => {
      const tree = renderer.create(<Title />).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it("Day", () => {
      const tree = renderer.create(<Day />).toJSON();
      expect(tree).toMatchSnapshot();
    });

    it("Activity", () => {
      const tree = renderer.create(<Activity />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });