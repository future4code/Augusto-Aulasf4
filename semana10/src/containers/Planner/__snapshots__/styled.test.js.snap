// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styleds Activity 1`] = `
<span
  className="sc-EHOje cDmsmP"
/>
`;

exports[`Styleds Day 1`] = `
<h3
  className="sc-ifAKCX gDVnEe"
/>
`;

exports[`Styleds IncludeButton 1`] = `
<button
  className="sc-bwzfXH bUFgUO"
/>
`;

exports[`Styleds InputsContainer 1`] = `
<div
  className="sc-htpNat jnQSKU"
/>
`;

exports[`Styleds PlannerContainer 1`] = `
<div
  className="sc-bdVaJa bjTQrD"
/>
`;

exports[`Styleds Title 1`] = `
<h3
  className="sc-bxivhb kirffd"
/>
`;
